packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -171,6 +171,141 @@ export type ScopedSchemaName<
 > = TScope extends undefined ? `${TName}` : `${TScope}.${TName}`;
 // > = `${TScope extends undefined ? "" : `${TScope}.`}${TName}`;
 
+/**
+ * Stateless APIs exposed via {@link SchemaFactory} as both instance properties and as statics.
+ * @privateRemarks
+ * We have no way to make linkable members which exist both as statics and instance properties since API-Extractor does not support this.
+ * As a workaround, we have this type as a third place which can be linked.
+ * @system @sealed @public
```

Comment:
Using them straight from here instead of `SchemaFactory.<member>` is ok for consumers? I was thinking if we should add `@remarks` here saying "don't use this, use the statics on `SchemaFactory` directly."

Reply:
It's not value exported (only type exported), so using them that way won't compile. It's also tagged `@system` so using this object directly isn't supported. I think that and the description pointing to SchemaFactory is sufficient.

Reply:
I missed the type-exported bit, that already makes it good enough, I think. Was just thinking of something more than the tag (since nothing enforces it), for people who don't pay attention to those.